src/ripple/shamap/SHAMap.h

```diff
@@ -182,6 +182,11 @@ class SHAMap
 
     void setUnbacked ();
 
+    bool isUnshared ()
```

Comment:
I don't think it's currently used anywhere and it could probably be removed. If not, it should certainly be made `const`.